### PR TITLE
 fix(component): card showcase profile

### PR DESCRIPTION
**Pull request by gepifanio**
Ticket: [#293](https://app.zenhub.com/workspaces/ffam-prototype-5c615588801d5a103031639c/issues/unimelb/ffam-prototype/293)

### CODE REVIEW

**Comment by gepifanio** [comment]
Comment: Deploy Started

Building with commit 2f034c17cc0efb7dab68aef587baef3789bb9a2b

https://app.netlify.com/sites/pattern-lib-unimelb/deploys/5cae99e0c3de3d0009d544cb

File: components/cards/CardShowcaseProfile.vue [hunk 1]
```diff
@@ -1,14 +1,16 @@
 <template>
   <div class="card card--bdr card--showcase-profile">
-    <img
-      :src="thumb"
-      :alt="title + ' image'"
-      class="card__image">
+    <div
+      v-if="thumb && thumb.length > 0"
+      :style="{ backgroundImage: `url(${thumb})` }"
+      class="card__image"
```
Comment (TimMansell) [inline components/cards/CardShowcaseProfile.vue]: We should refactor to utilise the `card__thumb` class being used on other card components

File: components/cards/_card-showcase-profile.css [hunk 2]
```diff
@@ -3,13 +3,17 @@
     padding: .5rem 0 0 1.5rem;
   }
 
-  .card__image {
-    width: 100%;
-    height: auto;
+  .card__thumb {
+    background-repeat: no-repeat;
```
Comment (TimMansell) [inline components/cards/_card-showcase-profile.css]: Don't need following styles as .card__thumb in cards.css sets it

background-repeat
background-position
background-size

File: components/cards/_card-showcase-profile.css [hunk 3]
```diff
@@ -3,13 +3,17 @@
...
+    background-position: center;
+    background-size: cover;
+    border-radius: 4px 0 0 4px;
```
Comment (TimMansell) [inline components/cards/_card-showcase-profile.css]: pretty sure we don't need border-radius

File: components/cards/CardShowcaseProfile.vue [hunk 4]
```diff
@@ -21,24 +23,32 @@
         />
       </button>
     </div>
+
     <transition name="slide-fade">
       <p
```
[the remark this answers is not in the document]
Comment (gepifanio) [reply components/cards/CardShowcaseProfile.vue]: Definitely true, I haven't realized that.

**Pull request merged**